src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.body.querySelector('ul');
+const listOfEmployeesChildrens = [...listOfEmployees.children];
+
+listOfEmployees.prepend(...sortList(listOfEmployeesChildrens));
+getEmployees(listOfEmployeesChildrens);
```

Comment:
The `getEmployees` function is called here, but its return value is not used. If the intention is to use the list of employee objects, consider storing the result in a variable.